app/controllers/content_types/modular_controller.rb

```diff
@@ -87,6 +87,120 @@ def one_column
     end
     # rubocop:enable Metrics/MethodLength
 
+    # GET /modular/two_column
+    # Maps to Content Types/Modular/2 Column in Cascade.
+    # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
+    def two_column
+      # Cascade Data Models
+      @configuration_set = ConfigurationSet.two_column
+      @metadata_set = MetadataSet.page(title: 'Modular Two Column')
+      @data_definition = DataDefinitions::TwoColumn.default
+
+      # Set dynamic values.
+      if params[:masthead] == 'branded-old'
+        @data_definition.set_value(:masthead_type, 'Branded Masthead')
+      elsif params[:masthead] == 'slider-old'
+        @data_definition.set_value(:masthead_type, 'Slider')
+      elsif params[:masthead] == 'slider'
+        @data_definition.set_value(:masthead_type, 'Slider - New')
+      else
+        @data_definition.set_value(:masthead_type, 'Branded - New')
+      end
```

Comment:
Well alright then, you know more about the situation. I just really really don't like behavior like this. It would already take a bit of time to refactor and make sure nothing broke, I just can't imagine what it would be like if we were refactoring these conditionals with 10/15/20 masthead options.